packages/visualizations/src/components/Table/Body.svelte

```diff
@@ -5,9 +5,17 @@
     export let loadingRowsNumber: number | null;
     export let columns: Column[];
     export let records: TableData;
+    export let emptyStateLabel = 'No records found...';
 </script>
 
 <tbody>
+    {#if records.length === 0}
+        <tr>
+            <td colspan={columns.length}>
+                <em>{emptyStateLabel}</em>
+            </td>
+        </tr>
+    {/if}
```

Comment:
We need to display this UI when `isLoading` is true
![image](https://github.com/opendatasoft/ods-dataviz-sdk/assets/117300300/8ed9ac24-5a3c-45e9-b1cd-71ebfc78a5d8)

Thanks to the loading story 😝